packages/react/ds-app-launchpad/src/ui/RelativeTime/RelativeTime.stories.tsx

```diff
@@ -0,0 +1,44 @@
+/* @canonical/generator-ds 0.9.0-experimental.4 */
+
+import { Temporal } from "@js-temporal/polyfill";
+import type { Meta, StoryFn, StoryObj } from "@storybook/react";
+import Component from "./RelativeTime.js";
+
+const meta = {
+  title: "",
+  component: Component,
+} satisfies Meta<typeof Component>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: StoryFn<typeof meta> = (args) => {
```

Comment:
It would be nice to either add a story for an invalid date or just add it as an additional date in this story, that way we can see what happens when the date is invalid.